test/WebSites/TagHelpersWebSite/Controllers/HomeController.cs

```diff
@@ -44,5 +44,10 @@ public ViewResult ViewWithInheritedRemoveTagHelper()
         {
             return View("/Views/RemoveTagHelperViewStart/ViewWithInheritedRemoveTagHelper.cshtml");
         }
+
```

Comment:
move this action and associated view (Environment.cshtml) into the _MVC_ tag helpers site -- test\WebSites\MvcTagHelpersWebSite